demo/text-field-custom-demos.html

```diff
@@ -0,0 +1,78 @@
+<dom-module id="text-field-custom-demos">
+  <template>
+    <style include="vaadin-component-demo-shared-styles">
+      :host {
+        display: block;
+      }
+    </style>
+
+    <h3>Custom Input</h3>
+    <p>
+      <b>Note:</b> In order to apply lumo or material theme styles to the placeholder
```

Comment:
There should preferably be some intro for the feature. Starting with NOTE is kind of unexpected.

Reply:
With intro I meant something like: "A native `<input>` element can optionally be defined separately as a child element for the `<vaadin-text-field>`. This will enable use-cases where having the `<input>` element available in the global DOM is necessary, such as supporting password managers or native forms.

The current intro, while valid, doesn't really tell the reader what "Custom Input" mentioned in the demo title means.

Reply:
There seems to be a nice intro in the `vaadin-text-field.html` docs already. Why not use that here also?